Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG GO_VERSION=1.20.11
-ARG VARIANT=bookworm
+ARG GO_VERSION=1.20.13
```

Comment:
```suggestion
ARG GO_VERSION=1.21.6
```

Reply:
I prefer not to mix these two changes. If anything fails, I want to be able to track what failed. I'll do it in a follow-up PR.